primitives/src/vid/advz.rs

```diff
@@ -866,9 +939,10 @@ mod tests {
 
             // 1 eval missing from all shares
             shares_missing_evals.last_mut().unwrap().evals.pop();
-            let bytes_recovered = advz
-                .recover_payload(&shares_missing_evals, &common)
-                .expect("recover_payload should succeed when shares have equal eval lengths");
+            let bytes_recovered = advz.recover_payload(&shares_missing_evals, &common).expect(
+                "recover_payload should succeed when shares have
+            equal eval lengths",
+            );
             assert_ne!(bytes_recovered, bytes_random);
```

Comment:
As discussed, this test is dumb. Please delete it.

Reply:
On second thought we should keep the test but check that payload recovery returns an error